docs/installing-solidity.rst

```diff
@@ -228,6 +228,28 @@ in Visual Studio 2017 Build Tools or Visual Studio 2017:
 .. _Visual Studio 2017: https://www.visualstudio.com/vs/
 .. _Visual Studio 2017 Build Tools: https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2017
 
+Prerequisites - OpenBSD
```

Comment:
The instructions here are extremely generic and do not look OpenBSD-specific. Perhaps rename this to "other" or something like that? :)

Reply:
Yeah, I agree @chriseth Our dependency scripts don't work on OpenBSD, so you do need these. I guess BSD is the most likely "other OS", but how about we use the language from above, something like "Other Unix-style OSs (e.g. BSD)", something like that? Just so the BSD part is searchable.

Reply:
I'm actually not sure we should encourage the use of the dependency script. Installing packages should not be taken lightly. Perhaps we should start the section about which dependencies are required and which are recommended in general and then go into detail about how exactly you do it on the specific systems.

Reply:
@chriseth Sounds like a good idea, but do you think it gets out of the scope of this PR?

Reply:
Yes, sounds like in scope for me, unless we have more detailed instructions for bsd